packages/gql-utils/prisma/__mocks__/prisma.ts

```diff
@@ -0,0 +1,25 @@
+import { beforeEach } from "vitest";
+import { mockDeep, mockReset } from "vitest-mock-extended";
+
```

Comment:
You may find https://github.com/thoughtbot/fishery useful. It is just a utility layer that provides factorios for creating test data.